### PR TITLE
Add product consideration to FIP-0070

### PR DESCRIPTION
**Pull request by steven004**

### CODE REVIEW

File: FIPS/fip-0070.md [hunk 1]
```diff
@@ -88,7 +88,14 @@ There might be a concern that overall _WindowPoSt_ messages in the network might
...
 
 ## Product Considerations
-There is no any impact on Filecoin ecosystem application or platforms. 
+The flexibility granted to Storage Providers (SPs) in controlling their WindowedPost Schedule allows them to establish maintenance windows during which they may not be available for data service. While this might appear to degrade service availability slightly compared to the current deadline assignment policy, it's important to consider the following factors, which limit the impact:
+
+1. **Data Availability for Winning Post**: The data being proven still needs to be available for mining (winning post). This incentivizes SPs to remain online and maintain data availability.
+2. **Separate Data Services**: In most cases, SPs provide services, especially for retrieval, from data that is separate from the sealed data. This can include fast retrieval, IPFS, Saturn, Webe.Storage or other web service systems.
+3. **Redundancy in Data Storag**e: In the Filecoin network, real data typically exists in multiple copies in the storage of different SPs. This redundancy ensures data availability even if one SP is temporarily unavailable.
```
Comment (anorth) [inline FIPS/fip-0070.md:95]:
```suggestion
3. **Redundancy in Data Storage**: In the Filecoin network, real data typically exists in multiple copies in the storage of different SPs. This redundancy ensures data availability even if one SP is temporarily unavailable.
```

**Review by anorth — APPROVED** [review]

**Review by Stebalien — APPROVED** [review]

File: FIPS/fip-0070.md [hunk 2]
```diff
@@ -88,7 +88,14 @@ There might be a concern that overall _WindowPoSt_ messages in the network might
 This FIP does not effect incentives in itself and is of only flexibility providing without any impact on economic factors. 
 
 ## Product Considerations
-There is no any impact on Filecoin ecosystem application or platforms. 
+The flexibility granted to Storage Providers (SPs) in controlling their WindowedPost Schedule allows them to establish maintenance windows during which they may not be available for data service. While this might appear to degrade service availability slightly compared to the current deadline assignment policy, it's important to consider the following factors, which limit the impact:
```
Comment (momack2) [inline FIPS/fip-0070.md:91]: a good thing to reference here is that these maintenance windows still require **_daily_** access and proving of **all** content (maintaining that key product requirement of provable storage - while lowering the burden on SPs to be constantly monitoring proving windows at any hour of the day or night)

File: FIPS/fip-0070.md [hunk 3]
```diff
@@ -88,7 +88,14 @@ There might be a concern that overall _WindowPoSt_ messages in the network might
...
+
+1. **Data Availability for Winning Post**: The data being proven still needs to be available for mining (winning post). This incentivizes SPs to remain online and maintain data availability.
```
Comment (momack2) [inline FIPS/fip-0070.md:93]: if an SP is not actively sealing new sectors (doing winning post), does this change the rational behavior to only have their node be online for a few minutes every 24hrs (ex if they move all their challenge windows to be the same epoch) -- or is it rational for all SPs to continue syncing the chain and staying online so as not to fall out of sync / risk missing their window post window?

Comment (anorth) [reply FIPS/fip-0070.md:93]: I think you have confused winning and window post. It's rational for a node to stay in sync and have storage online in order to be able to meet winning post if they win the block election.

**Review by kaitlin-beegle — APPROVED** [review]

**Pull request merged**